app/lib/frontend/models.dart

```diff
@@ -239,6 +241,50 @@ abstract class SecretKey {
   ];
 }
 
+/// A random ID, typically sent via e-mail, to verify and confirm an action.
+///
+/// [action] identifies the trigger and scope of the verification. It ensures
+/// that the given verification instance can't be reused for different purpose
+/// (e.g. filling the [parameters] for one goal and then trigger an action in
+/// another part of the application for another goal).
+@db.Kind(name: 'Verification', idType: db.IdType.String)
```

Comment:
Can we call this entity a `UrlSecret` instead?

It's a secret that we put into the URL, isn't that right?

Better names are also welcome :)